tdrs-backend/tdpservice/users/api/utils.py

```diff
@@ -36,25 +36,25 @@
 """
 
 
-def validate_nonce_and_state(decoded_nonce, state, nonce_validator, state_validator):
+def validate_nonce_and_state(request, state, decoded_id_token):
     """Validate nonce and state are correct values."""
+    # get the validation keys to confirm generated nonce and state
+    nonce_and_state = get_nonce_and_state(request.session)
+    nonce_validator = nonce_and_state.get("nonce", "not_nonce")
+    state_validator = nonce_and_state.get("state", "not_state")
+
+    decoded_nonce = decoded_id_token["nonce"]
+
     return decoded_nonce == nonce_validator and state == state_validator
 
 
 """
 Generate the client_assertion parameter needed by the login.gov/token endpoint
-
-:param self: parameter to permit django python to call a method within its own class
 """
 
 
 def generate_client_assertion():
-    """
-    Generate client assertion parameters.
-
-    :param JWT_KEY: private key expected by the login.gov application
-    :param CLIENT_ID: Issuer as defined login.gov application
```

Comment:
Should these be re-added back in or are they inaccurate now?

Reply:
This is inaccurate now